### PR TITLE
NYT

### PR DESCRIPTION
**Pull request by Alaska01**
<h1>Project: Positioning and Floating Elements</h1> 

#### What does this PR do?
created NYT clone 
#### Description of Task to be completed?
Create a new HTML document.
Think about all the elements on the page and how they are grouped together.
Layout the basic structure of the page using HTML semantic, HTML elements that are appropriately sized and positioned. 
Set up the top navigation bar 
Stick the navbar to the top of the page as you scroll down
Fill in and position the additional sections.
Used a combination of grid and flex to carry out the design.
#### How should this be manually tested?
just clone the project and open the index.html file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,12 +3,762 @@
...
+                    <h2>How to See Comet SWAN in Night Skies</h2>
+                    <p>May 12</p>
+                </div>
+                <div class="ppl-item">
+                    <img src="https://static01.nyt.com/images/2020/05/06/science/06SCI-STARLINK/06SCI-STARLINK-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale"
+                        alt="item2" />
+                    <h3>Peter Komka/EPA, via Shutterstock</h3>
+                    <h2>
+                        SpaceX Plans Sunshades to Save Night Skies From Starlink
+                        Satellites
```
Comment (lucasmazo32) [inline index.html]: - [ ] Please make the text (just title) and images links, just like the original site.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

I have one strongly advice. Please make more descriptive commits. Debugging and debugging again are not descriptive. I know how frustrating it is, but please try to limit the number of commits as much as you can because it doesn't look professional. 

### Happy coding! :computer:

## Lucas Mazo

File: index.html (hunk 1 above)
Comment (Alaska01) [reply index.html]: Thanks for the review, the links have been added to the images and text as requested. Your comment on making descriptive commits have been noted and utilized in the changes made. Bravo!!!

**Review by sumancrest0001 — APPROVED** [review]
**STATUS: APPROVED :100: 
Congratulation TEAM :clap: :clap: :+1:** 

You have done an incredible job to make it similar to the original layout. Everything looks good to me including sticky navbar and footer. Alignments of items are great. It's a YES from my side. I request you to make one change though. **Editor's pick** title is just a link and there is no styling to it. You can add some styling to give it the original look.
mission


**Happy coding**

**Comment by mujohn26** [comment]
> **STATUS: APPROVED 💯 Congratulation TEAM 👏 👏 👍**
> 
> You have done an incredible job to make it similar to the original layout. Everything looks good to me including sticky navbar and footer. Alignments of items are great. It's a YES from my side. I request you to make one change though. **Editor's pick** title is just a link and there is no styling to it. You can add some styling to give it the original look.
> mission
> 
> **Happy coding**

Thanks @sumancrest0001

**Pull request merged**